src/core/server/http/http_server.test.ts

```diff
@@ -1291,30 +1384,6 @@ test('should return a stream in the body', async () => {
   });
 });
 
-test('closes sockets on timeout', async () => {
-  const { registerRouter, server: innerServer } = await server.setup({
-    ...config,
-    socketTimeout: 1000,
-  });
-  const router = new Router('', logger, enhanceWithContext);
-
```

Comment:
@mshustov I basically just copy pasted the modification you did for this file in https://github.com/elastic/kibana/pull/95384. Not sure if I should keep this removal?

Reply:
yeah, that looks like an irrelevant change, mind reverting?